package.json

```diff
@@ -1,4 +1,94 @@
 {
   "name": "builder-react-component",
-  "version": "0.0.0"
+  "version": "0.0.1",
+  "description": "Builder Archetype - React Component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/builder-react-component.git"
+  },
+  "author": "Ryan Roemer <ryan.roemer@formidablelabs.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/builder-react-component/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/builder-react-component",
+  "scripts": {
+    "npm:postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || builder run build-lib",
+    "npm:preversion": "builder run check",
+    "npm:version": "builder run clean && builder run build && git add -A dist",
+    "npm:test": "builder run test-frontend",
+    "clean-dist": "rimraf dist",
+    "build-dist-min": "webpack --config node_modules/builder-react-component/config/webpack/webpack.config.js",
```

Comment:
That's pretty much the _whole point_ of an archetype: In the common case you do nothing and it works.

If you _do_ want to override, what you do is:

1. In your `package.json` add a `build-dist-min` script command pointing to a local `ROOT/webpack.config.js`
2. That `webpack.config.js` can either:
    1. `require("builder-react-component/config/webpack/webpack.config.js")` and mutate / extend; OR
    2. Rewrite it from scratch.

And then you've "made it your own"